src/Orchard.Cms.Web/Modules/Orchard.Demo/Shapes/BazTagHelper.cs

```diff
@@ -1,9 +1,18 @@
 ﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+using Orchard.DisplayManagement;
+using Orchard.DisplayManagement.TagHelpers;
 
-namespace Orchard.Demo.Shapes
+namespace Orchard.Demo.TagHelpers
 {
-    public class BazTagHelper : TagHelper
+    [HtmlTargetElement("baz", Attributes = nameof(Text))]
```

Comment:
That's actually wrong. The intent of this class was not to have a "functional" tag helper, but one that would hold the documentation/intellisense part. The actual behavior is actually provided by the `ShapeTagHelperDescriptorResolver` which will create as many tag helper instances as there are shape descriptors.